include/swupd-build-variant.h

```diff
@@ -27,4 +36,4 @@
 #define TAR_PERM_ATTR_ARGS "--preserve-permissions "
 #endif
 
-#endif
+#endif /* SWUPD_WITH_BSDTAR */
```

Comment:
This `#endif` closes the include guard, not SWUPD_WITH_BSDTAR. Can you move the comment to the correct spot?